File: Python Code/Daily_Flash_Code_0/Week13/17Apr/76-DailyFlash_Solutions/17_April_Solution_Two/Java/Program0.java
import java.io.*;

class Program1 {


	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		System.out.printf("Enter A String 1\n");
		String str1 = new String(br.readLine());

		System.out.printf("Enter A String 2\n");
		String str2 = new String(br.readLine());

		if(str1.length() != str2.length()){

			System.out.println("Strings are Unequal");
		}
		else{

			int cmp = 0;

			for(int i = 0; i < str1.length(); i++){

				if(str1.charAt(i) != str2.charAt(i)){

					cmp = str1.charAt(i) - str2.charAt(i);
					break;
				}
			}

			if(cmp == 0){

				System.out.println("Strings are equal");
			}
			else {

				System.out.println("Strings are Unequal and Differ by "+cmp);
			}


		}

	}
}

File: Python Code/Daily_Flash_Code_0/Week13/17Apr/76-DailyFlash_Solutions/17_April_Solution_Two/CPP/Program0.cpp
#include <stdio.h>

int cmp(char* str1,char* str2){

	while( *str1 != '\n' || *str2 != '\n'){

		if(*str1 != *str2){

			return (*str1 - *str2);
		}
		str1++;
		str2++;
	}
	return 0;
}

int main() {

	char inp1[100] = {'\0'};
	char inp2[100] = {'\0'};

	printf("Input a string 1\n");
	fgets(inp1,sizeof(inp1),stdin);

	printf("Input a string 2\n");
	fgets(inp2,sizeof(inp2),stdin);

	int cm = cmp(inp1,inp2);

	if(cm == 0){

		printf("Strings are Equal\n");
	}
	else{

		printf("String differ by %d\n",cm);
	}

	return 0;
}
